x-pack/plugins/apm/public/components/shared/KueryBar/view.js

```diff
@@ -39,17 +39,25 @@ class KueryBarView extends Component {
     isLoadingSuggestions: false
   };
 
-  willUnmount = false;
+  mounted = false;
+  id = null;
 
   componentWillUnmount() {
-    this.willUnmount = true;
+    this.mounted = false;
   }
 
   async componentDidMount() {
+    this.mounted = true;
+    this.id = new Date().getTime();
```

Comment:
Is this left-over, or where is `this.id` used?

Reply:
Oops, left over debug. Thanks!